MANIFEST.in

```
@@ -5,7 +5,7 @@ include requirements-docs.txt
 global-include *.pyi
 
 # For mnemonics
-recursive-include eth_account/hdaccount/wordlist/*
+recursive-include eth_account hdaccount wordlist *.txt
```

Aha good find! By my reading of [this](https://packaging.python.org/guides/using-manifest-in/):

> recursive-include dir-pattern pat1 pat2 ... | Include all files under directories matching dir-pattern that match any of the listed patterns

I would expect that the intended approach is:
```suggestion
recursive-include eth_account/hdaccount/wordlist *.txt
```

Does that work?

---

Also, it's worth making sure you have the latest tooling with a `pip install -U pip setuptools wheel`. They have seemed pretty active.